src/now.js

```diff
@@ -517,6 +520,13 @@ const main = async argv_ => {
   try {
     exitCode = await commands[subcommand](ctx);
   } catch (err) {
+    if (err.code === 'ENOTFOUND' && err.hostname === 'api.zeit.co') {
+      output.error('You are offline. Please ensure your device has an active internet connection.');
```

Comment:
-1. We can't absolutely determine the user is offline. Let's be more technically accurate.

> `api.zeit.co` could not be resolved. Please verify your internet connectivity or DNS configuration

Reply:
Fixed